browser/extensions/brave_extension_provider.cc

```diff
@@ -126,7 +127,8 @@ bool BraveExtensionProvider::UserMayLoad(const Extension* extension,
 
 bool BraveExtensionProvider::MustRemainInstalled(const Extension* extension,
                                                  base::string16* error) const {
-  return extension->id() == brave_extension_id;
+  return extension->id() == brave_extension_id ||
```

Comment:
should this be here? Do we want it always loaded by default or on user activation?

Reply:
I would prefer loaded by default for now if that's OK.
And address it in a follow-up issue to make it able to be turned on and off by user at anytime.

AFAIK, if we want to disable a component extension from UI, it should probably go through chrome://extensions.
First, to see component extensions listed, we'll need to pass --show-component-extension-options when run it.
But it seems I can't toggle the on&off switch even if the component extension is shown for our current version.
I would like to address this in a separate issue.
@bridiver wdyt?

Reply:
btw, it seems it is still loaded by default even when I remove this line.

Reply:
this doesn't affect default loading, it only affects the ability to uninstall